.golangci.yml

```diff
@@ -23,7 +23,6 @@ linters:
     - ineffassign
     - misspell
     - staticcheck
-    - structcheck
```

Comment:
have you checked for `//nolint:structcheck` comments?

(perhaps enable the `nolintlint` linter as well, which should catch those)